helm/hpcc/templates/_helpers.tpl

```diff
@@ -693,15 +709,6 @@ securityContext:
 Generate instance queue names
 */}}
 {{- define "hpcc.generateConfigMapQueues" -}}
-{{- $storage := ($.Values.storage | default dict) -}}
-{{- $planes := ($storage.planes | default list) -}}
-{{- $firstPlane := dict -}}
-{{- range $plane := $planes -}}
- {{- if and (not $firstPlane.plane) (or (not $plane.labels) (has "data" $plane.labels)) }}
- {{- $_ := set $firstPlane "plane" $plane.name -}}
- {{- end -}}
-{{- end -}}
-{{- $defaultDataPlane := ($storage.dataStorage.plane | default $firstPlane.plane | default "hpcc-data-plane") -}}
```

Comment:
better to keep this line and assign the default, rather than evaluating each time.  (I think the parameters to default are evaluated regardless of whether they are needed or not.)